presto-main/src/main/java/io/prestosql/memory/RemoteNodeMemory.java

```diff
@@ -120,7 +120,7 @@ public void onSuccess(@Nullable JsonResponse<MemoryInfo> result)
                             memoryInfo.set(Optional.ofNullable(result.getValue()));
                         }
                         if (result.getStatusCode() != OK.code()) {
-                            log.warn("Error fetching memory info from %s returned status %d: %s", memoryInfoUri, result.getStatusCode(), result.getStatusMessage());
```

Comment:
but why? it seems the message is almost always terse and only helps to understand the status code, but it can also carry some more information

Reply:
We are removing them from Airlift since they aren’t in HTTP/2 and aren’t well supported on the server side (so it’s not even possible to test portably).